framework/moose.mk

```diff
@@ -41,7 +41,7 @@ pyhit_srcfiles  := $(hit_DIR)/hit.cpp $(hit_DIR)/lex.cc $(hit_DIR)/parse.cc $(hi
 #
 # FParser JIT defines
 #
-ADDITIONAL_CPPFLAGS += -DADFPARSER_INCLUDES="\"-I$(FRAMEWORK_DIR)/build/header_symlinks -I$(LIBMESH_DIR)/include\""
+ADDITIONAL_CPPFLAGS += -DADFPARSER_INCLUDES="\"-I$(FRAMEWORK_DIR)/framework/include/utils -I$(FRAMEWORK_DIR)/framework/include/base -I$(LIBMESH_DIR)/include\""
```

Comment:
Oh you need to remove framework from the paths here

Reply:
Just a test of your review-prowess

Reply:
Well I'd give myself a C since I missed it the first time haha. I'm too trusting of you. It's gotten us in a lot of trouble recently haha